src/index.ts

```diff
@@ -1,9 +1,16 @@
 import { Application } from 'probot' // eslint-disable-line no-unused-vars
 
+const welcomeMessage: string = `Thank you for contributing to Leapp project!
+
+Please note that every PR needs to comply with the [Leapp Guidelines](https://github.com/oamg/leapp-guidelines), pass tests and linter check before it can be merged.
+If you want to re-run tests or request review, you can use following commands as a comment:
+- "leapp-ci build" - runs unit tests
+- "e2e test" - runs unit tests and end-to-end tests
```

Comment:
Just regarding the recent discussion, people outside of oamg group will not be able to run those tests. From that point, shouldn't be here information like (oamg group only) or drop it completely?